ui/projekt-monsterz-front/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "projekt-monsterz-front",
```

Comment:
To keep the nomenclature in line, consider changing the package name to 'projekt-monsterz-ui'